gcc/config/riscv/riscv-vector-builtins-bases.h

```diff
@@ -109,6 +109,10 @@ extern const function_base *const vwmacc;
 extern const function_base *const vwmaccu;
 extern const function_base *const vwmaccsu;
 extern const function_base *const vwmaccus;
+extern const function_base *const sf_vqmacc;
+extern const function_base *const sf_vqmaccu;
+extern const function_base *const sf_vqmaccsu;
+extern const function_base *const sf_vqmaccus;
```

Comment:
Could you create a new file `sifive-vector-builtins-bases.h` to hold sifive intrinsic